mantidimaging/core/operations/remove_dead_stripe/remove_dead_stripe.py

```diff
@@ -13,18 +13,19 @@
 
 
 class RemoveDeadStripesFilter(BaseFilter):
-    """Algorithm 6 in the paper. Remove unresponsive or fluctuating stripes by:
-    locating stripes, correcting by interpolation. Angular direction is along
-    the axis 0.
+    """Stripe and ring artifact removal. Algorithm 6 in Vo et al., Optics Express 28396 (2018).
+    Remove unresponsive or fluctuating stripes by: locating stripes, correction by interpolation.
+    Angular direction is along the axis 0.
```

Comment:
What is axis 0? Will the user know what this is?

Reply:
I'll remove the "axis 0" sentence. Its from the sarepy docs, and useful for developers not MI users.